commonv5/ts/sonarqube/Scanner.ts

```diff
@@ -135,7 +137,17 @@ export class ScannerCLI extends Scanner {
   }
 
   public async runAnalysis() {
-    let scannerCliScript = tl.resolve(this.rootPath, "sonar-scanner", "bin", "sonar-scanner");
+    // TODO: instead of relying on bundled version, lets download it before this task
+    const scannerLocation: string = tl.getVariable(TaskVariables.SonarQubeScannerLocation);
```

Comment:
This technically can be `string | undefined` (undefined when neither cliVersion nor msBuildVersion is provided in prepare-task)